### DISCUSSION TITLE
Goals for 2024

### DISCUSSION THREAD

**Discussion by AOrthVector** [genevaers/community#264] [Ideas]
Ideas for 2024 goals that will be added to the end of year recap

**Comment by AOrthVector** [comment]
The road to gold

- Achieve Silver badge
-

**Comment by AOrthVector** [comment]
Finalize the future of MR88

**Reply by ghannington** [reply]
We will aim to create a more streamlined (easier to maintain) version of the format routine GVBMR88. 
Consider supporting new functions such as COUNT and AVERAGE (MEAN).

**Comment by AOrthVector** [comment]
GitHub project management

**Comment by neilbeesley** [comment]
Maybe add support to access Adabas data also, like we did in the read exit.
